### PR TITLE
Update readme re: LESS and Django Compressor

### PR DESCRIPTION
**Pull request by biyeun**
@benrudolph or anyone

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -181,16 +181,134 @@ that you have a 32bit version of Python installed.
...
+COMPRESS_MTIME_DELAY = 3  # set to higher or lower depending on how often you're editing static files
+COMPRESS_REBUILD_TIMEOUT = 6000
+```
+
+###### Install LESS
+
+*Note:* The reason why we have TWO versions, rather than one is that the newest
+LESS that Twitter Bootstrap 3.0 runs off of is not backwards compatible with
+LESS 1.3.1, which Twitter Bootstrap 2.3.2 needs. Since we're running BOTH
+simultaneously, we need to have two versions installed.
```
Comment (benrudolph) [inline README.md]: ouch

File: README.md [hunk 2]
```diff
@@ -181,16 +181,134 @@ that you have a 32bit version of Python installed.
...
+```
+
+Option 3 is really only useful if you're trying to debug issues that mirror
+production that's related to staticfiles and compressor. For all practical uses
+please use Option 2 or Option 1 to save yourself the headache.
+
+#### CloudCare
 
 To enable CloudCare, ensure that `TOUCHFORMS_API_USER` and
 `TOUCHFORMS_API_PASSWORD` in `localsettings.py` are the credentials of the
```
Comment (benrudolph) [inline README.md]: this is sweet thanks

**Comment by czue** [comment]
awesome!

File: README.md (hunk 1 above)
Comment (esoergel) [inline README.md]: Yeah, but when are we switching to SASS and Bootstrap 4?

Comment (biyeun) [inline README.md]: No. Those bastards.

File: README.md [hunk 3]
```diff
@@ -181,16 +181,134 @@ that you have a 32bit version of Python installed.
...
+Take note of this variable already in `settings.py`
+```
+LESS_FOR_BOOTSTRAP_3_BINARY = '/opt/lessc/bin/lessc'
+```
+
+You can change that to wherever you clone the git repo for 1.7.3, or leave it
+as is and follow this accordingly:
+
+    1. in `/opt`: `git clone https://github.com/less/less.js.git lessc`
+    2. In the `lessc` repo `git reset HEAD --hard 546bedd3440ff7e626f629bef40c6cc54e658d7e`
```
Comment (proteusvacuum) [inline README.md]: I think this shouldn't have `HEAD` here

**Comment by proteusvacuum** [comment]
This literally saves seconds on every page load locally for me. :tada:

File: README.md (hunk 3 above)
Comment (biyeun) [inline README.md]: Oops, I got a little aHEAD of myself

Comment (esoergel) [inline README.md]: ba_dum_tss_pirates_band_of_misfits

**Pull request merged**